mod/item.php

```diff
@@ -698,6 +698,8 @@ function item_post(App $a) {
 		Item::setHashtags($datarray);
 	}
 
+	$datarray["body"] = Item::fixPrivatePhotos( $datarray["body"], $datarray["uid"], $datarray, 0 );
```

Comment:
Code standards: Please remove the spaces inside parentheses. See https://www.php-fig.org/psr/psr-2/#46-method-and-function-calls